service/chartconfig/v1/helm/helm.go

```diff
@@ -122,7 +122,11 @@ func (c *Client) InstallFromTarball(path, ns string, options ...helmclient.Insta
 
 // UpdateReleaseFromTarball updates the given release using the chart packaged
 // in the tarball.
-// TODO
 func (c *Client) UpdateReleaseFromTarball(releaseName, path string, options ...helmclient.UpdateOption) error {
+	_, err := c.helmClient.UpdateRelease(releaseName, path, options...)
+	if err != nil {
```

Comment:
Why do we need two commands doing the same thing?

Reply:
The method here is a proxy for the dependency method, we can model an interface around it and pass mocks implementing it for testing.

Reply:
Also, this way we don't need to make the internal helm client public to provide functionality from it.